.pre-commit-config.yaml

```diff
@@ -46,10 +46,10 @@ repos:
         name: check-added-large-files
       - id: check-shebang-scripts-are-executable
         name: check-shebang-scripts-are-executable
-        exclude: ^(wandb/|tests/)
+        exclude: ^(wandb/|tests/|landfill/)
```

Comment:
lol

Reply:
tried to think of a name that would easy to grep